### ISSUE TITLE
0 columns does weird wrapping

### ISSUE THREAD

**Issue by marknelissen** [xenomachina/kotlin-argparser#17]
When not setting the columns value, the help output is weirdly formatted, since it behaves as if you try to wrap with a very small value, instead of no wrap as the javadoc indicates.

Sample code
```
class Configuration(parser: ArgParser) {
    val test by parser.storing(help = "lol test")
    init {
        parser.force()
    }
}
fun main(args: Array<String>) = mainBody(columns = 0) {
    Configuration(ArgParser(args))
}
```

Output:
```
usage: 
        
        [
        -
        h
        ]
        
        -
        -
        t
        e
        s
        t
        
        T
        E
        S
        T

required arguments:
     
  -  l
  -  o
  t  l
  e  
  s  t
  t  e
     s
  T  t
  E
  S
  T


optional arguments:
     
  -  s
  h  h
  ,  o
     w
  -  
  -  t
  h  h
  e  i
  l  s
  p  
     h
     e
     l
     p
     
     m
     e
     s
     s
     a
     g
     e
     
     a
     n
     d
     
     e
     x
     i
     t
```

This has been obtained with kotlin 1.1.2-2 on jre 8.

**Comment by xenomachina** [comment]
I can reproduce this. Thanks for reporting it.

**Comment by xenomachina** [comment]
Ok, this should be fixed in release 2.0.1
